tests/v1/entrypoints/llm/test_struct_output_generate.py

```diff
@@ -15,16 +15,36 @@
 from vllm.outputs import RequestOutput
 from vllm.sampling_params import GuidedDecodingParams, SamplingParams
 
+NGRAM_SPEC_CONFIG = {
+    "model": "[ngram]",
+    "num_speculative_tokens": 5,
+    "prompt_lookup_max": 5,
+    "prompt_lookup_min": 1,
+},
+
+EAGLE_SPEC_CONFIG = {
+    "method": "eagle",
+    "model": "abhigoyal/EAGLE-LLaMA3-Instruct-8B-vllm",
+    "num_speculative_tokens": 5,
+}
+
 PARAMS_MODELS_BACKENDS_TOKENIZER_MODE = [
     ("mistralai/Ministral-8B-Instruct-2410", "xgrammar:disable-any-whitespace",
-     "auto"),
+     "auto", None),
     ("mistralai/Ministral-8B-Instruct-2410", "guidance:disable-any-whitespace",
-     "auto"),
+     "auto", None),
     ("mistralai/Ministral-8B-Instruct-2410", "xgrammar:disable-any-whitespace",
-     "mistral"),
-    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar:disable-any-whitespace", "auto"),
-    #FIXME: This test is flaky on CI thus disabled
-    #("Qwen/Qwen2.5-1.5B-Instruct", "guidance:disable-any-whitespace", "auto"),
+     "mistral", None),
+    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar:disable-any-whitespace", "auto",
+     None),
```

Comment:
It looks like you re-enabled this test case by accident?

Reply:
This was a while ago, but I think that we have made improvements to how we handle disable-any-whitespace. The test passes locally for me consistently, and if the CI is passing on the PR then I think it might be safe to keep. 

Do you have any additional context regarding this?

Reply:
reverted.